pkgs/development/python-modules/generic/default.nix

```diff
@@ -129,6 +129,11 @@ python.stdenv.mkDerivation (builtins.removeAttrs attrs ["disabled" "doCheck"] //
   # Python packages don't have a checkPhase, only an installCheckPhase
   doCheck = false;
 
+  passthru = {
```

Comment:
Good start on the passthru; there are a few more things I think we should put in there but this is good :+1: